tower_cli/models/base.py

```diff
@@ -115,10 +115,11 @@ def __new__(cls, name, bases, attrs):
 
         # Ensure that the endpoint ends in a trailing slash, since we
         # expect this when we build URLs based on it.
-        if not newattrs['endpoint'].startswith('/'):
-            newattrs['endpoint'] = '/' + newattrs['endpoint']
-        if not newattrs['endpoint'].endswith('/'):
-            newattrs['endpoint'] += '/'
+        if isinstance(newattrs['endpoint'], basestring):
```

Comment:
It looks like some python environment hit `./tower_cli/models/base.py:118:45: F821 undefined name 'basestring'`. I think that jangsutsr was working on a solution to that problem, and he's probably figured it out by now.